### PR TITLE
Add a blog:  Overview of Plugin

### PR DESCRIPTION
**Pull request by t-kikuc**
**What this PR does**:
as title.
I translated from my japanese post https://zenn.dev/cadp/articles/pipecd-plugin-intro.

**Why we need it**:

Let's announce the plan and get feedback from developers.

### CODE REVIEW

**Review by khanhtc1202 — APPROVED** [review]
LGreatTM 💯

File: docs/content/en/blog/plugin-intro.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+For the current PipeCD users, please migrate to the Pluginnable version.  New features will be added to the Pluginnable version.
+We will share the details of migration ways later.  We've been making efforts to minimize the amount of the migration tasks. The main tasks will be modifying Piped configs and Application configs.
+
+## Conclusion
+
+Pluginnable PipeCD is a big step toward the PipeCD's vision - "**The One CD for All {applications, platforms, operations}**".  The community energy will be more important.
+
+To see the development progress, visit [the RFC](https://github.com/pipe-cd/pipecd/blob/master/docs/rfcs/0015-pipecd-plugin-arch-meta.md) or [the meta Issue](https://github.com/pipe-cd/pipecd/issues/5259).
+
+If you have any questions or opinions, feel free to talk in [CNCF Slack](https://cloud-native.slack.com/) > #pipecd-jp or [PipeCD Community Meeting](https://docs.google.com/document/d/1AtE0CQYbUV5wLfvAcl9mo9MyTCH52BuU7AngVUvE7vg/edit).
```
Comment (Warashi) [inline docs/content/en/blog/plugin-intro.md:125]: [nits]
It's perhaps better to note on a non-Japanese channel.

```suggestion
If you have any questions or opinions, feel free to talk in [CNCF Slack](https://cloud-native.slack.com/) > #pipecd or [PipeCD Community Meeting](https://docs.google.com/document/d/1AtE0CQYbUV5wLfvAcl9mo9MyTCH52BuU7AngVUvE7vg/edit).
```

**Review by Warashi — COMMENTED** [review]
Great! thank you.
I commented on one nitpick.

File: docs/content/en/blog/plugin-intro.md (hunk 1 above)
Comment (t-kikuc) [reply docs/content/en/blog/plugin-intro.md]: @Warashi 
Thank you! I fixed in [7979946](https://github.com/pipe-cd/pipecd/pull/5372/commits/79799461b1836de20b763df756e62ccf3a5dfd2c)

**Review by khanhtc1202 — APPROVED** [review]

**Review by Warashi — APPROVED** [review]
Thanks

**Pull request merged**